metadata.json

```diff
@@ -103,7 +103,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.5.2 < 5.0.0"
```

Comment:
Preferably 4.6.1 because yes it works better (mostly around create_resources bugs like https://tickets.puppetlabs.com/browse/PUP-6230), but the 4.5 series was the first that fixed data-in-modules (https://tickets.puppetlabs.com/browse/PUP-6230). I'll check with the gang.